login-status.bs

```diff
@@ -1,11 +1,10 @@
 <pre class="metadata">
-Title: The IsLoggedIn API
-Shortname: is-logged-in
+Title: The Login Status API
+Shortname: login-status
 Repository: privacycg/is-logged-in
 URL: https://privacycg.github.io/is-logged-in/
 Editor: John Wilander, w3cid 89478, Apple Inc. https://apple.com/, wilander@apple.com
-Editor: Melanie Richards, w3cid 96078, Microsoft https://microsoft.com/, Melanie.Richards@microsoft.com
```

Comment:
Instead of removing this line, prepend "Former " and Bikeshed will do the right thing.